### PR TITLE
APIGOV-19476 Remove meshv1 instructions

### PR DESCRIPTION
**Pull request by tsjohns9**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Updating the mesh management docs to remove references to the old meshv1 install instructions since meshv1 is being deprecated.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by tsjohns9** [comment]
@lbadenhop @andreamussap Do these changes look good?

File: content/en/docs/central/mesh_management/deploy-your-agents-with-the-axway-cli.md [hunk 1]
```diff
@@ -6,14 +6,12 @@ date: 2020-12-0
...
 
-{{< alert title="Public beta" color="warning" >}}This is a preview of the new Istio agents, which run separately from previous Istio POC agents that provide full governance of your hybrid environment. The new agents are deployed and configured from the Axway CLI, and they monitor Kubernetes resource discovery and Istio traffic activity.{{< /alert >}}
-
 ## Before you begin
 
 Ensure you have the following tools installed:
 
 * Axway Central CLI 1.7.0 or later
-* Helm 3.2.4 or later
+* Helm 3.2.4 or later jeroen
```
Comment (lbadenhop) [inline content/en/docs/central/mesh_management/deploy-your-agents-with-the-axway-cli.md:14]:
```suggestion
* Helm 3.2.4 or later jeroen  (Is this correct?)
```

**Review by lbadenhop — APPROVED** [review]
@tsjohns9 See my comment

File: content/en/docs/central/mesh_management/deploy-your-agents-with-the-axway-cli.md (hunk 1 above)
Comment (tsjohns9) [reply content/en/docs/central/mesh_management/deploy-your-agents-with-the-axway-cli.md]: fixed

**Pull request merged**